libraries/chain/account_evaluator.cpp

```diff
@@ -125,13 +125,6 @@ void_result account_create_evaluator::do_evaluate( const account_create_operatio
       FC_ASSERT( !op.extensions.value.owner_special_authority.valid() );
       FC_ASSERT( !op.extensions.value.active_special_authority.valid() );
```

Comment:
Please add a comment why HARDFORK_516 has to stay, to simplify future cleanups.

Reply:
TBH, I don't see why it needs to stay. Re-evaluating (adding log messages) to see if it can be removed.

Reply:
Ran a snapshot for mainnet, and grep'd logs with no trace of 516 changing anything. Removing.